Makefile

```diff
@@ -11,6 +11,7 @@ NODE ?= node
 ZUUL ?= $(NODE) $(BIN)/zuul
 
 test:
+	$(BIN)/mocha test/*.js; \
```

Comment:
Actually, I've never edited a Makefile before, so this syntax may or may not be entirely correct.